argusclient/client.py

```diff
@@ -362,6 +362,7 @@ def update(self, id, obj):
         if id != obj.argus_id:
             raise ValueError("Object id: %s doesn't match the id: %s that you are updating" % (obj.id, id))
         self._coll[id] = self.argus._request("put", self.id_path % id, dataObj=obj)
+        self._fill(self._coll[id])
```

Comment:
Since `_fill` returns the object, is slightly safer to capture the object after passing it through this method:
```suggestion
        self._coll[id] = self._fill(self.argus._request("put", self.id_path % id, dataObj=obj))
```